### DISCUSSION TITLE
Favicon Fingerprinting - What To Do About It?

### DISCUSSION THREAD

**Discussion by BugHunter1000** [Eloston/ungoogled-chromium#1388] [Q&A]
https://old.reddit.com/r/netsec/comments/ldyova/demo_abusing_favicons_for_fingerprinting/

Does UC have a solution to this the way Firefox does?

Thanks.

**Comment by jstkdng** [comment]
What's firefox's solution for this? I guess sending a GET request regardless of the f-cache could solve this.

**Comment by kramred** [comment]
This will most likely be fixed in chromium, see [here](https://bugs.chromium.org/p/chromium/issues/detail?id=1116371&q=1096660&can=2) and issue 1096660 (once that is available without security clearance)

Firefox is not loading favicons from cache due to a [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1618257), which turned out to be a feature in this case of fingerprinting…

This seems to be kind of a hot potato right now as the scientific publication by [Jason Polakis](https://www.cs.uic.edu/~polakis/) has also been (temporarily) removed – although there is an proof of concept (describing the method) here: https://supercookie.me/workwise

**Reply by kramred** [reply]
This behaviour is bound to your profile (mainly the SQLite 3.x database file `Favicons`), so switching profiles will probably give you different fingerprints. Browsing as Guest user will give you a new fingerprint each time you start a Guest browsing session.
